modules/pam_namespace/pam_namespace.c

```diff
@@ -1379,6 +1381,9 @@ static int inst_init(const struct polydir_s *polyptr, const char *ipath,
 					/* ignore failures, they don't matter */
 				}
 
+				if (close_range(3, ~0U, 0) < 0)
+					_exit(1);
```

Comment:
I doubt we can assume that `close_range` is universally available.
Let's use `pam_modutil_sanitize_helper_fds` instead, e.g.
```
if (pam_modutil_sanitize_helper_fds(idata->pamh, PAM_MODUTIL_IGNORE_FD,
                                    PAM_MODUTIL_IGNORE_FD,
                                    PAM_MODUTIL_IGNORE_FD) < 0)
        _exit(1);
```
```

Reply:
Do you have any guidelines regarding backwards compatibility? `pam_namespace` is Linux specific so other operating systems should not matter. `close_range()` has been available since kernel 5.9 which is about 3 years old now.

Ah I see that `pam_modutil_sanitize_helper_fds()` tries to use `close_range()` internally, if autoconf finds it. All right, I'll switch to the helper function then.

Reply:
I pushed a rebased version using `pam_modutil_sanitize_helper_fds()`